pytype/tools/analyze_project/main.py

```diff
@@ -74,6 +74,9 @@ def main():
   logging.info('Source tree:\n%s',
                importlab.output.formatted_deps_list(import_graph))
   tool_utils.makedirs_or_die(conf.output, 'Could not create output directory')
+  f = open(conf.output + ".gitignore", "w")
```

Comment:
This will create a file called ".pytype.gitignore" (assuming the output dir is .pytype/). If you use `os.path.join(conf.output, '.gitignore')`, that should create ".pytype/.gitignore", which is what you want.